ephyviewer/datasource/neosource.py

```diff
@@ -110,63 +110,6 @@ def get_sources_from_neo_segment(neo_seg):
 
 ## neo.rawio stuff
 
-# this can be remove when neo version 0.10 will be out
-class AnalogSignalFromNeoRawIOSource_until_v9(BaseAnalogSignalSource):
-    def __init__(self, neorawio, channel_indexes=None):
-
-        BaseAnalogSignalSource.__init__(self)
-        self.with_scatter = False
-
-        self.neorawio =neorawio
-        if channel_indexes is None:
-            channel_indexes = slice(None)
-        self.channel_indexes = channel_indexes
-        self.channels = self.neorawio.header['signal_channels'][channel_indexes]
-        self.sample_rate = self.neorawio.get_signal_sampling_rate(channel_indexes=self.channel_indexes)
-
-        #TODO: something for multi segment
-        self.block_index = 0
-        self.seg_index = 0
-
-    @property
-    def nb_channel(self):
-        return len(self.channels)
-
-    def get_channel_name(self, chan=0):
-        return self.channels[chan]['name']
-
-    @property
-    def t_start(self):
-        t_start = self.neorawio.get_signal_t_start(self.block_index, self.seg_index,
-                    channel_indexes=self.channel_indexes)
-        return t_start
-
-    @property
-    def t_stop(self):
-        t_stop = self.t_start + self.get_length()/self.sample_rate
-        return t_stop
-
-    def get_length(self):
-        length = self.neorawio.get_signal_size(self.block_index, self.seg_index,
-                        channel_indexes=self.channel_indexes)
-        return length
-
-    def get_gains(self):
-        return self.neorawio.header['signal_channels']['gain'][self.channel_indexes]
-
-    def get_offsets(self):
-        return self.neorawio.header['signal_channels']['offset'][self.channel_indexes]
-
-    def get_shape(self):
-        return (self.get_length(), self.nb_channel)
-
-    def get_chunk(self, i_start=None, i_stop=None):
-        sigs = self.neorawio.get_analogsignal_chunk(block_index=self.block_index, seg_index=self.seg_index,
-                        i_start=i_start, i_stop=i_stop, channel_indexes=self.channel_indexes)
-        return sigs
-
-
-# this fit the neo API >= 0.10 (with streams concept)
 class AnalogSignalFromNeoRawIOSource(BaseAnalogSignalSource):
     def __init__(self, neorawio, channel_indexes=None, stream_index=None):
```

Comment:
Maybe we should add something here in docstring to remember that 
 * for neo<0.10 `channel_indexes` is absolut
 * for neo>=0.10 `channel_indexes` is relative to stream

Reply:
Another good idea.